Documentation/Contributors/CodingGuide/README.md

```diff
@@ -117,6 +117,14 @@ A few more naming conventions are introduced below along with their design patte
 - For HTML code, keep the existing style. Use double quotes.
 - Text files, end with a newline to minimize the noise in diffs.
 
+## Spelling
+
+- We have a basic setup for `cspell` to spellcheck our files. This is currently not enforced but recommended to use and check while programming. This is especially true for JSDoc comments that well end up in our documentation or Readme files
+  - Run `npm run cspell` to check all files
+  - Run `npx cspell -c .vscode/cspell.json [file path]` to check a specific file
```

Comment:
```suggestion
- Run `npm run cspell -c .vscode/cspell.json [file path]` to check a specific file

Reply:
This was intentional and I think it should stay this way. the `npm run cspell` command encapsulates how to run it for the whole project. It's the same as the `prettier` and `eslint` scripts. if people want to run it for specific files they can do it this way but I don't assume that will be the normal intention.